react-native-cli/index.js

```diff
@@ -188,11 +188,11 @@ function createProject(name, verbose) {
 }
 
 function run(root, projectName) {
-  exec('npm install --save react-native', function(e, stdout, stderr) {
+  exec('npm install --save react react-native', function(e, stdout, stderr) {
```

Comment:
might be worth moving `npm install --save react` to `local-cli`? This way existing users won't have to upgrade global `react-native-cli`